docs/variants/msi_z690/firmware-update.md

```diff
@@ -37,3 +162,56 @@ In this case, the whole `bios` region must be updated.
 ```bash
 flashrom -p internal -w [path] --ifd -i bios
 ```
+
+## Troubleshooting
```

Comment:
This section is very generic, shouldn't it go to: https://docs.dasharo.com/osf-trivia-list/deployment/#flashrom ?

Reply:
Right... I have moved it to the suggested section